src/metabase/db/liquibase.clj

```diff
@@ -345,6 +349,25 @@
   [s]
   (map #(Integer/parseInt %) (re-seq #"\d+" s)))
 
+(defn- record-legacy-changelogs!
+  "Create records for all the legacy migrations that were aggregated, so that we will not attempt to re-run them."
+  [conn]
+  (let [liquibase-table-name (changelog-table-name conn)]
+    (when-not (some-> (jdbc/query {:connection conn} [(str "SELECT ID FROM " liquibase-table-name " WHERE ID = '1'")]) seq)
+      (with-open [rdr (io/reader (io/resource "migrations/000_legacy_migrations.yaml"))]
+        (let [legacy-migrations (->> rdr yaml/parse-stream :databaseChangeLog (keep :changeSet))
+              insert-statement  (format "INSERT INTO %s (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, EXECTYPE) VALUES (?, ?, ?, CURRENT_TIMESTAMP, ?, ?)" liquibase-table-name)
```

Comment:
I'm surprised that md5sum is nullable :/. wonder if we should put ANY in just to be sure?

Reply:
lol @ liquibase 🥲

Reply:
I think NULL is the most honest for a phantom 🤷👻